src/algorithms/dqns/dqn.jl

```diff
@@ -53,6 +55,7 @@ function DQNLearner(;
     traces = SARTS,
     update_step = 0,
     rng = Random.GLOBAL_RNG,
+    double::Bool = true
```

Comment:
A more detailed name would be better here, like `is_enable_double_DQN`

Reply:
Yes, I will change the name.